### ISSUE TITLE
ZeroDivisionError: integer division or modulo by zero

### ISSUE THREAD

**Issue by williballenthin** [fireeye/capa#704]
```
   1   │ Traceback (most recent call last):
   2   │   File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
   3   │     return _run_code(code, main_globals, None,
   4   │   File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
   5   │     exec(code, run_globals)
   6   │   File "/home/user/code/capa-pub/capa/main.py", line 991, in <module>
   7   │     sys.exit(main())
   8   │   File "/home/user/code/capa-pub/capa/main.py", line 899, in main
   9   │     extractor = get_extractor(
  10   │   File "/home/user/code/capa-pub/capa/main.py", line 464, in get_extractor
  11   │     vw = get_workspace(path, format, sigpaths)
  12   │   File "/home/user/code/capa-pub/capa/main.py", line 412, in get_workspace
  13   │     vw = viv_utils.getWorkspace(path, analyze=False, should_save=False)
  14   │   File "/home/user/env/lib/python3.8/site-packages/viv_utils-0.6.5-py3.8.egg/viv_utils/__init__.py", line 101, in getWorkspace
  15   │     vw.loadFromFile(fp)
  16   │   File "/home/user/env/lib/python3.8/site-packages/vivisect/__init__.py", line 2676, in loadFromFile
  17   │     fname = mod.parseFile(self, filename, baseaddr=baseaddr)
  18   │   File "/home/user/env/lib/python3.8/site-packages/vivisect/parsers/pe.py", line 36, in parseFile
  19   │     return loadPeIntoWorkspace(vw, pe, filename, baseaddr=baseaddr)
  20   │   File "/home/user/env/lib/python3.8/site-packages/vivisect/parsers/pe.py", line 180, in loadPeIntoWorkspace
  21   │     secrem = len(header) % secalign
  22   │ ZeroDivisionError: integer division or modulo by zero
```

**Comment by williballenthin** [comment]
need either:
  - viv to handle this case robustly, or
  - a more specific bug from viv that we and handle and format

**Comment by williballenthin** [comment]
closed in #731
